cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/export/CgmesExportContextTest.java

```diff
@@ -67,22 +54,4 @@ void emptyConstructor() {
         assertTrue(context.exportBoundaryPowerFlows());
         assertEquals("1D", context.getBusinessProcess());
     }
-
-    @Test
-    void getSet() {
```

Comment:
Would this test be relevant with CIM16 for example? I mean should we delete it completely or use a CIM16 version?